### DISCUSSION TITLE
Real World Data Science - ‘What if we try to bring R to the classroom? That was our wacky idea’

### DISCUSSION THREAD

**Discussion by giscus** [realworlddatascience/realworlddatascience.github.io#175] [General]
# Real World Data Science - ‘What if we try to bring R to the classroom? That was our wacky idea’

R-Girls is a new project aiming to introduce R into secondary school lessons – just not through computer science classes! Real World Data Science meets with one of the project’s founders, Mohammed Mohammed, to learn more.

https://realworlddatascience.net/viewpoints/interviews/posts/2023/09/25/rgirls-interview.html

**Comment by samrit2442** [comment]
This is indeed a great initiative.
